content/tutorial/03-sveltekit/03-loading-data/02-layout-data/README.md

```diff
@@ -47,4 +47,4 @@ Now, add a sidebar in the layout for the post page:
 
 The layout (and the page below it) inherits `data.summaries` from the parent `+layout.server.js`.
 
-When we navigate from one post to another, we only need to load the data for the post itself — the layout data is still valid. See the documentation on [invalidation](https://kit.svelte.dev/docs/load#invalidation) to learn more.
+When we navigate from one post to another, we only need to load the data for the post itself — the layout data is still valid. See the documentation on [invalidation](https://kit.svelte.dev/docs/load#rerunning-load-functions-manual-invalidation) to learn more.
```

Comment:
This should link to the information on when `load` reruns instead of how to use `invalidate`

```suggestion
When we navigate from one post to another, we only need to load the data for the post itself — the layout data is still valid. See the documentation on [invalidation](https://kit.svelte.dev/docs/load#rerunning-load-functions) to learn more.
```